internal/pkg/template/templates/addons/aurora/env/rdws/ingress.yml

```diff
@@ -0,0 +1,28 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: Your workload's name.
+  ServiceSecurityGroupId:
+    Type: String
+    Description: The security group associated with the VPC connector.
+Resources:
+  {{logicalIDSafe .ClusterName}}ClusterSecurityGroupIngressFromWorkload:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      Description: !Sub Ingress from workload ${Name}.
+      ToPort: {{- if eq .Engine "MySQL" }} 3306 {{- else}} 5432 {{- end}}
+      FromPort: {{- if eq .Engine "MySQL" }} 3306 {{- else}} 5432 {{- end}}
+      GroupId:
+        Fn::ImportValue: !Sub "${App}-${Env}-{{logicalIDSafe .ClusterName}}DBClusterSecurityGroup"
+      IpProtocol: tcp
+      SourceSecurityGroupId: !Ref ServiceSecurityGroupId
+Outputs:
+  {{logicalIDSafe .ClusterName}}AuroraServerlessSecret: # injected as {{ logicalIDSafe .ClusterName | printf "%sAuroraServerlessSecret" | envVarSecret | toSnakeCase}} environment variable by Copilot.
+    Description: "The JSON secret that holds the database username and password. Fields are 'host', 'port', 'dbname', 'username', 'password', 'dbClusterIdentifier' and 'engine'"
+    Value: { Fn::ImportValue: { Fn::Sub: "${App}-${Env}-{{logicalIDSafe .ClusterName}}AuroraSecret" }}
```

Comment:
Interesting 🤔 

Shouldn't the customer use:
```yaml
secrets:
   DB_SECRET:
      secretsmanager:
         from_cfn: ${COPILOT_APP_NAME}-${COPILOT_ENV_NAME}-ClusterNameAuroraSecret
```

If I'm understanding correctly, the output in this PR will ony inject the ARN of the secretsmanager secret as an env var and the user will have to fetch the secret within the container

Reply:
I think with @yohanthshetty's recent change, the injection is possible now 🤔 https://github.com/aws/copilot-cli/blob/a60cf0fbac6a339dc6e98019258fa70159a9359b/internal/pkg/template/templates/workloads/services/rd-web/cf.yml#L98

Reply:
My brain might not be working, iirc `NestedStack.SecretOutputs` are discovered as SecretsManager secrets by searching for the logical ID and checking the Type of the resource. Since this is an import, we won't be able to find that 🤔 

Let me know if I'm missing something!

Reply:
good call boss it's my brain that is not working 😂 

I am going to remove this `Output` for this PR. In one of the next PR that handles `RecommendedActions`, I will recommend a step to inject the secret from manifest:
```
Recommended follow-up actions:
- Run `copilot env deploy` to deploy your environment addons, along with the environment. 
- Update the manifest for your `api` service:
```
secrets:
  USER_DB_SECRET:
    from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-UserDBSecret
```
```